apps/dashboard/src/components/primitives/control-input/control-input.tsx

```diff
@@ -67,247 +44,19 @@ export function ControlInput({
   indentWithTab,
   isAllowedVariable,
 }: ControlInputProps) {
-  const viewRef = useRef<EditorView | null>(null);
```

Comment:
moved all this code to the `VariableEditor` that is reused between the `ControlInput` and `HtmlEditor`